app/assets/javascripts/components/select_experience_domain.js

```diff
@@ -35,18 +41,28 @@ class SelectExperienceDomain extends React.PureComponent<Props, State> {
   }
 
   render() {
+    const options = this.getUnusedDomains().map(domain => <Option key={domain}>{domain}</Option>);
+    const notFoundContent = this.props.notFoundContent || "Not Found";
```

Comment:
I think this variable is not necessary. If it's undefined, antd will default to "Not found" automatically.